sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/administration/models/ModelOperationDetails.java

```diff
@@ -5,6 +5,7 @@
 
 
 import com.azure.ai.formrecognizer.implementation.util.ModelOperationDetailsHelper;
+import com.azure.core.models.ResponseError;
```

Comment:
@alzimmermsft/ @srnagar Do we know why we do not allow setting properties of `details`, `innerError` or `target` on the [ResponseError](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core/src/main/java/com/azure/core/models/ResponseError.java) model in core?

Reply:
We had an [arch board discussion](https://github.com/Azure/azure-sdk/issues/3221) on this topic. The gist of the discussion is that only the message and error code is considered API contract. All other details like inner error, target etc. are not part of the API contract and can change in ways that can break backward compatibility. So, the response error type in azure-core only exposes message and code. 

The recording is in the issue if you are interested in the full discussion.